imap_upstream.adoc

```diff
@@ -31,4 +31,156 @@ The available balancing algorithms are as follows:
       custom:{handler-algorithm} [arg1 arg2 ...]
 
 [NOTE]
-If the custom load balancing class can't be located the default ClientIpHash will be returned.
+If the custom load balancing class can't be located the default
+ClientIpHash will be returned.
+
+= IMAPD-related Configuration & Files
+
+This section summarizes all of the configuration options related to
+the `zimbra-imapd` as well as all other files that are installed for
+the new service.
+
+== Global Configuration Settings
+
+* These control the embedded IMAP(S) servers that are run inside of
+  `mailboxd`.
+** `zimbraImapServerEnabled`
+** `zimbraImapBindPort`
+** `zimbraImapSSLServerEnabled`
+** `zimbraImapSSLBindPort`
+* These control the IMAP(S) servers that are run by `zimbra-imapd`.
+** `zimbraRemoteImapServerEnabled`
+** `zimbraRemoteImapBindPort`
+** `zimbraRemoteImapSSLServerEnabled`
+** `zimbraRemoteImapSSLBindPort`
+* This maintains the list of available IMAP(S) servers:
+** `zimbraReverseProxyUpstreamImapServers`
+
+== Server Configuration Settions
+
+If the `zimbra-imapd` service is enabled on a given server, it will be added to
+the multi-valued attribute `zimbraServiceEnabled`.
+
+For example, on an IMAP-only server:
+
+[EXAMPLE]
+----
+$ zmprov gs `zmhostname` zimbraServiceEnabled
+# name HOST.DOMAIN
+zimbraServiceEnabled: imapd
+zimbraServiceEnabled: stats
+----
+
+== Files that are installed for zimbra-imapd
+
+These are the files that are specifically installed by the
+zimbra-imapd DEB/RPM:
+
+* `/opt/zimbra/bin/zmimapdctl`
+* `/opt/zimbra/conf/imapd.log4j.properties`
+* `/opt/zimbra/lib/jars/oauth-1.4.jar`
+
+In addition, the service makes use of other jar files that are already
+installed into `/opt/zimbra/lib/jars` via the `zimbra-core` package.
+
+== Files that are created during installation
+
+`zmcertmgr` is invoked by `zmsetup.pl` to create the following
+SSL-related files:
+
+* `imapd.crt`
+* `imapd.key`
+* `imapd.keystore`
+
+== Log files
+
+The following log files (in `/opt/zimbra/log`) are created by the
+`zimbra-imapd` service via configuration contained in
+`/opt/zimbra/conf/imapd.log4j.properties`.
+
+* `imapd-audit.log`
+* `imapd.log`
+
+
+== Process Files
+
+The following files are created directly by the `zmimapdctl` script:
+
+* `imapd.out`
+* `imapd.pid`
+
+== Other Files
+
+Activity statistics are collected in `/opt/zimbra/zmstat/imapd.csv`.
+
+
+= Migrating to an imapd pool
+
+When migrating to dedicated pool of IMAP servers, several steps must
```

Comment:
Thank you!  This has been corrected in a fix-up commit.  I did not regenerate the HTML for the `gh-pages` view yet.